activities/value-and-impact/user-mapping/management-and-procurement.md

```diff
@@ -0,0 +1,52 @@
+---
+type: Resource
+---
+
+# 
```

Comment:
This is an empty title ;)

Given how it is linked from the index it should probably read:

```suggestion
# Business management and procurement for public administrations
```